test/unit_tests/modules/nav-primary-spec.js

```diff
@@ -87,45 +87,48 @@ describe( 'Get Event States', function() {
 
   describe( 'Primary Nav Link Events - One expanded', function() {
     beforeEach( function() {
-      $link = $( $( '.sub-nav-2' ).find( 'a' )[0] );
+      link = document.querySelector( '.sub-nav-2 a' );
```

Comment:
Is it worth it to create a shortcut like `var  docSel document.querySelector.bind(document)`?